packages/api-client/src/notification/NotificationAPI/NotificationAPI.ts

```diff
@@ -108,6 +108,10 @@ export class NotificationAPI {
             hasMissedNotifications = true;
             payload = {...defaultPayload, ...error.response?.data};
           }
+        } else if (error instanceof BackendError && error.label === BackendErrorLabel.NOT_FOUND) {
```

Comment:
older API's (e.g. `v2`) should not be affected since they are always `AxiosError` type and contain body with `notifications`